tauri-app/src/lib/test/settings.test.ts

```
@@ -1,22 +1,38 @@
+  activeAccountsItems,
+  activeSubgraphs,
+  activeNetworkRef,
+  activeOrderbookRef,
+  resetActiveNetworkRef,
+  resetActiveOrderbookRef,
+  activeNetworkOrderbooks,
+  hasRequiredSettings,
+  subgraphUrl,
+  accounts,
+  activeAccounts,
+  activeOrderStatus,
```

ESLint

[error] 'activeOrderStatus' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)